openfe/setup/box_representation.py

```diff
@@ -0,0 +1,104 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+import numpy as np
+
+
+def is_valid_box(box: np.ndarray) -> bool:
+    """Check that a box is valid
+
+    Current rules:
+    - the y and z component of the x vector must be zero
+    - the z component of the y vector must be zero
+
+    Parameters
+    ----------
+    box : np.ndarray, shape(3,3)
+       the simulation cell
+
+    Returns
+    -------
+    valid : bool
+    """
+    if box.shape != (3, 3):
+        return False
+    lx, xy, xz = box[0]
+    yx, ly, yz = box[1]
+    zx, zy, lz = box[2]
+
+    if (xy != 0.0) or (xz != 0.0) or (yz != 0.0):
+        return False
+    # TODO: Max tilt factors, are these common across all MD engines?
```

Comment:
I think that's just a LAMMPS thing right?

Reply:
I think OpenMM strictly requires this, too.

Reply:
And gmx - https://gitlab.com/gromacs/nb-lib/-/blob/nblib/src/gromacs/pbcutil/pbc.cpp#L166